lib/common.js

```diff
@@ -250,6 +250,21 @@ exports.createTableForApps = function (apps) {
   return table;
 };
 
+exports.createTableForSubmissions = function(submissionsResult){
+  var table = new Table({
+    head: ['ID', 'Form ID', 'Form Name', 'Project GUID', 'Project Name', 'Submitted At'],
```

Comment:
Just a nit-pick: the heading says 'Submitted At' but you insert a time difference into the table (fromNow). I would expect a timestamp (or maybe rename to column heading to just 'Submitted'?)